### DISCUSSION TITLE
RepRapFirmware Support.

### DISCUSSION THREAD

**Discussion by P-C-R** [OctoFarm/OctoFarm#241] [Feature Request]
RRF got a pretty easy API to call! Would be nice!

**Comment by NotExpectedYet** [comment]
It's planned along with marina and klipper but thanks for starting the discussion. Would there be any reprap specific settings that would be handy to have?

**Comment by NotExpectedYet** [comment]
Please see: https://github.com/OctoFarm/OctoFarm/discussions/1298
